#! /bin/sh /usr/share/dpatch/dpatch-run
## DP: Fixes assembly version

@DPATCH@

--- cecil/Mono.Cecil/Mono.Cecil/AssemblyInfo.cs.old	2008-03-20 18:20:55.000000000 +0100
+++ cecil/Mono.Cecil/Mono.Cecil/AssemblyInfo.cs	2008-03-20 18:21:01.000000000 +0100
@@ -43,7 +43,7 @@
 [assembly: CLSCompliant (false)]
 [assembly: ComVisible (false)]
 
-[assembly: AssemblyVersion ("0.6.0.0")]
+[assembly: AssemblyVersion ("0.5.0.0")]
 
 #if KEYFILE
 [assembly: AssemblyKeyFile("../../mono.snk")]
